contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -125,21 +145,30 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
         // Burn native token by sending to BURN_FOR_TRANSFER_ADDRESS
         Address.sendValue(payable(BURN_FOR_TRANSFER_ADDRESS), value);
 
+        uint256 scaledAmount;
+        if (multiplyOnReceive) {
+            scaledAmount = msg.value / tokenMultiplier;
```

Comment:
I assume that integer division should never be an issue (i.e., rounding down) for any reasonable amount and multipler values.

Reply:
Yeah, in the worst case you could lose up to the smallest divisible unit on either of the chains, which is fine.